assets/js/admin/network-orders.js

```diff
@@ -0,0 +1,90 @@
+/*global woocommerce_network_orders */
+(function( $, _, undefined ) {
+
+	if ( 'undefined' === typeof woocommerce_network_orders ) {
+		return;
+	}
+
+	var orders = [],
+		promises = [], // Track completion (pass or fail) of ajax requests.
+		deferred = [], // Tracks the ajax deferreds.
+		$tbody = $( document.getElementById( 'network-orders-tbody' ) ),
+		template = _.template( $( document.getElementById( 'network-orders-row-template') ).text() ),
+		$loadingIndicator = $( document.getElementById( 'woocommerce-network-order-table-loading' ) ),
+		$orderTable = $( document.getElementById( 'woocommerce-network-order-table' ) ),
+		$noneFound = $( document.getElementById( 'woocommerce-network-orders-no-orders' ) );
```

Comment:
Any reason for using ID rather than jQuery based selectors? You have class and ID set in the markup - just asking :)

Reply:
@mikejolley  Class was set to enable styling that wasn't too specific. 

`getElementById` was used for performance reasons over the jQuery selectors. `document.getElementsByClassName` is faster than `getElementById`, but isn't supported on as many browsers (IE11+) whereas `getElementById` is supported on basically all browsers even back to IE6. jQuery selectors are slower than all of the above options. 

![Perf Test](https://cldup.com/1zTOLq6tly.png)